corehq/apps/locations/templates/locations/location_types.html

```diff
@@ -157,15 +158,17 @@ <h2>{% trans "Organization Levels" %}</h2>
                                           {% endblocktrans %}">
                                     </span>
               </th>
-              <th>
-                {% trans "View Child Data to Level" %}
-                <span class="hq-help-template"
-                      data-title="{% trans "View Child Data to Level" %}"
-                      data-content="{% blocktrans %}
-                          <p>Only include cases that show up in the user's restore file down to this level (inclusive).</p>
-                      {% endblocktrans %}">
-                </span>
-              </th>
+              {% if location_case_sync_restriction_enabled %}
```

Comment:
You can actually do this in the template without needing to pass anything into context
```suggestion
              {% if request|toggle_enabled:"USH_RESTORE_FILE_LOCATION_CASE_SYNC_RESTRICTION" %}
```

Reply:
Yeah, that's my rationale - whichever requires less code.  Certainly not wrong either way though.